pippo-core/src/main/java/ro/pippo/core/Request.java

```diff
@@ -277,6 +278,19 @@ public String getContextUri() {
     }
 
     /**
+     * Returns the uri relative to the application base uri.
+     *
+     * @return the uri relative to the application base url
+     */
+    public String getApplicationUri() {
```

Comment:
I don't "feel" the method name. I must read the javadoc to understand what this method do. I don't know if `getRelativeUri` is a better name